npm/install.js

```diff
@@ -0,0 +1,82 @@
+const fs = require("fs");
+const https = require("https");
+const path = require("path");
+
+const packageJson = require("./package.json");
+const { version, name, repository } = packageJson;
+
+async function install() {
+  const downloadURL = getDownloadURL();
+
+  try {
+    const binDir = path.join(__dirname, "bin");
+    const exeName = ["win32", "cygwin"].includes(process.platform)
+      ? `${name}.exe`
```

Comment:
When the Go CLI is compiled against windows, does it automatically add the `.exe` extension? It doesn't get that automatically when I cross compile on linux.

Reply:
The released windows binaries didn't have the `.exe` extension. So the install script is handling this.
https://github.com/Anush008/pizzaroni-cli/blob/6b21cb84f88f75467ce6f270e136dfca5e462d23/npm/install.js#L13-L18
https://github.com/Anush008/pizzaroni-cli/blob/6b21cb84f88f75467ce6f270e136dfca5e462d23/npm/install.js#L65
By appending the extension whenever it is downloaded from a win platform.